doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -140,9 +144,13 @@ Testing steps
 
 To perform a FOTA update, complete the following steps:
 
-1. Create a binary file that contains the new image.
+.. fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
 
-   |fota_upgrades_building|
+1. Generate the DFU package by building your application with the FOTA support over Bluetooth Low Energy.
+   You can find the generated DFU archive in the following location :file:`<build_dir>/zephyr/dfu_application.zip`.
```

Comment:
```suggestion
   You can find the generated DFU archive in the following directory :file:`<build_dir>/zephyr/dfu_application.zip`.
```

Reply:
Well, we passed the full path to the file. To be more precise, I suggest the following:

```suggestion
   You can find the generated :file:`dfu_application.zip` archive in the following directory :file:`<build_dir>/zephyr`.
```

How does it sound to you, @annwoj?

Reply:
looks good!